pkg/controlplane/controller/kubernetesservice/controller.go

```diff
@@ -153,20 +165,6 @@ func (c *Controller) Run(ch <-chan struct{}) {
 // UpdateKubernetesService attempts to update the default Kube service.
 func (c *Controller) UpdateKubernetesService(reconcile bool) error {
 	// Update service & endpoint records.
-	// TODO: when it becomes possible to change this stuff,
-	// stop polling and start watching.
-	// TODO: add endpoints of all replicas, not just the elected master.
-	if _, err := c.client.CoreV1().Namespaces().Get(context.TODO(), metav1.NamespaceDefault, metav1.GetOptions{}); err != nil {
```

Comment:
this is handled here now https://github.com/kubernetes/kubernetes/blob/4af23c157c3416dbbbd34c94204959c564322a59/pkg/controlplane/controller/systemnamespaces/system_namespaces_controller.go#L49-L51